docs/sources/setup-grafana/image-rendering/_index.md

```diff
@@ -2,11 +2,11 @@
 aliases = ["/docs/grafana/latest/administration/image_rendering/", "/docs/grafana/latest/image-rendering/"]
 description = "Image rendering"
 keywords = ["grafana", "image", "rendering", "plugin"]
-title = "Image rendering"
-weight = 55
+title = "Set up image rendering"
+weight = 900
 +++
 
-# Image rendering
+# Set up image rendering
```

Comment:
Since the other topics are "Configure ..." and we are already in a Setup category, should we reanem this to "Configure image rendering" for consistency, even though it involves a little bit more than just config

Reply:
I see there are some other pages prefixed with "Set up" so not a solid argument :)